SimPEG/EM/FDEM/SrcFDEM.py

```diff
@@ -435,14 +481,16 @@ def s_eDeriv(self, prob, v, adjoint=False):
 
                 return -C.T * (MMui_sDeriv * v)
 
-            elif formulation == 'HJ':
-                return Zero()
+            elif prob._formulation == 'HJ':
+                if adjoint is True:
+                    return np.zeros_like(prob.model)
+                return np.zeros(prob.mesh.nF)
                 # raise NotImplementedError
-                mu_s = prob.mu - self.mu
-                MMui_s = prob.mesh.getEdgeInnerProduct(mu_s, invMat=True)
-                C = prob.mesh.edgeCurl.T
+                # mu_s = prob.mu - self.mu
```

Comment:
These lines were not hit previously. I would like to leave them as comments for thinking through this problem